docs/pull-requests.rst

```diff
@@ -0,0 +1,73 @@
+Preview Documentation from Pull Requests
```

Comment:
We should probably just call these Pull Requests in the content here to match the title, instead of `Pull/Merge requests`. I think everyone knows what PR's are at this point.

Reply:
+1 on that